### ISSUE TITLE
latest tensorflow

### ISSUE THREAD

**Issue by raybellwaves** [pangeo-data/pangeo-docker-images#243]
FWIW

I used latest tensorflow in a project recently with

```
channels:
   - conda-forge
   - nvidia
dependencies:
 - cudatoolkit
 - cuda
 - cudnn
 - pip:
   - tensorflow
   - git+https://github.com/tensorflow/docs
```

until https://github.com/conda-forge/tensorflow-feedstock/pull/131 goes in

**Comment by scottyhq** [comment]
Thanks @raybellwaves  we used https://github.com/pangeo-data/pangeo-docker-images/blob/75b4f2add543b4c5a74012a846e0cc7877ebdac0/ml-notebook/environment.yml#L14 to have GPU support. Not sure who has been using the ML image though and if it is even working as expected. But in order to have everything from conda-forge I think this is the issue to keep an eye on https://github.com/conda-forge/tensorflow-feedstock/issues/10

**Comment by scottyhq** [comment]
closed by #261 (images > 2021.11.04)
